timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -236,6 +152,46 @@ def set_contexts(self, path_context):
         self.path = path_context
         self.path_root = self.path.rsplit(self.name, 1)[0]
 
+    @classmethod
+    def check_and_split_hyperparameters(
+        cls,
+        hyperparameters: Optional[Dict[str, Any]] = None,
+        ag_args_fit_key: str = AG_ARGS_FIT,
+    ) -> Tuple[Dict[str, Any], Dict[str, Any]]:
+        """
+        Given the user-specified hyperparameters, split into `params` and `extra_ag_args`.
```

Comment:
This statement and the `Returns` are a bit out of date since we don't use the name `params` anymore.